assets/js/util/when-active.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * whenActive HOC.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { createElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as MODULES_STORE } from '../googlesitekit/modules/datastore/constants';
```

Comment:
Please use the same naming convention for the alias here as we do in other places 🙂 
```suggestion
import { STORE_NAME as CORE_MODULES } from '../googlesitekit/modules/datastore/constants';
```